Develop issue 1740 (#5298)
magento/devdocs: _includes/config/setup-cron_2.2_how-to.md

To create the Magento crontab:

1.	Log in as, or switch to, the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html).
1.	Change to your Magento installation directory.
1.	Enter the following command:

    ```bash
    bin/magento cron:install [--force]
    ```

Use `--force` to rewrite an existing Magento crontab.

{: .bs-callout-info }
*	`magento cron:install` does not rewrite an existing crontab inside `#~ MAGENTO START` and `#~ MAGENTO END` comments in your crontab.
*	`magento cron:install --force` has no effect on any cron jobs outside the Magento comments.

To view the crontab, enter the following command as the Magento file system owner:

```bash
crontab -l
```

A sample follows:

```terminal
#~ MAGENTO START

> * * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run | grep -v Ran jobs by schedule >> /var/www/html/magento2/var/log/magento.cron.log

Reviewer: There are some things missing here, should be:
`* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run 2>&1 | grep -v "Ran jobs by schedule" >> /var/www/html/magento2/var/log/magento.cron.log`

Notice the `2>&1` and the double quotes in the grep command.

* * * * * /usr/bin/php /var/www/html/magento2/update/cron.php >> /var/www/html/magento2/var/log/update.cron.log
* * * * * /usr/bin/php /var/www/html/magento2/bin/magento setup:cron:run >> /var/www/html/magento2/var/log/setup.cron.log
#~ MAGENTO END
```

{%
include note.html
type='info'

> content='`update/cron.php` does exist if you install Magento using [Composer](https://glossary.magento.com/composer), or [download the archive](https://magento.com/tech-resources/download), in both cases the file exists.

Reviewer 2: ```suggestion
content='The `update/cron.php` file exists if you install Magento using [Composer](https://glossary.magento.com/composer) or by [downloading the archive](https://magento.com/tech-resources/download).
```

Reviewer 2: It seems like what we're really trying to say here is that the `update/cron.php` file **does not exist** if you installed Magento by cloning a git repo. Is that right? If so, then I propose simplifying the paragraph:

> The `update/cron.php` file exists in Composer- and archive-based Magento installations. It does not exist if you installed Magento by cloning a git repository.

Author: **implemented**

> content='The `update/cron.php` file exists in [Composer](https://glossary.magento.com/composer)- and [archive-based](https://magento.com/tech-resources/download) Magento installations. It does not exist if you installed Magento by cloning a git repository.

Reviewer: This `a git repository` is a bit too generic. It should mention cloning [the Magento2 repo](https://github.com/magento/magento2/). But not sure how to phrase this exactly?

> In the case of [Composer](https://glossary.magento.com/composer), the directory `update` and all the files in there are only created when you run `composer create-project`, however if you only run `composer install` you will not see the `update` directory getting created (if it did not exist before). Check [Recreate the Magento updater]({{ site.url }}/guides/v2.3/comp-mgr/cli/cli-upgrade.html#recreate-magento-updater) topic to get an info how to solve the issue with the not existing `update` folder.'

Reviewer 2: ```suggestion
In [Composer-based installations](https://glossary.magento.com/composer), Magento creates the `update/` directory when you run `composer create-project`. Running `composer install` does not create the `update/` directory (if it did not exist before). See [Recreate the Magento updater](https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html#recreate-magento-updater).
```

Reviewer 2: Unfortunately, you cannot use liquid tags (`{{ site.url }}`) in this type of note format. It throws a liquid error:

```
Liquid Exception: Invalid syntax for include tag:
```

Author: **fixed**
